build.gradle

```diff
@@ -0,0 +1,55 @@
+//
+// Objectives:
+//
+// - The galasactl tool needs to embed the galasa-boot.jar so that the jar can be 
+//   unpacked on disk and used to launch testcases in a local jvm.
+//   So we need to get hold of this boot jar.
+//
+// - The galasactl tool needs to talk to the API server via an HTTP client layer, 
+//   which is generated from a yaml definition file.
+//   So we need to get hold of a generator tool so we can 
+//
+
+def galasaBootJarVersion = '0.24.0'
```

Comment:
should this be updated?

Reply:
Ideally we'd have a property file which the build generates, and the tool would draw such values from that.

It might be better if it was bumped, but not necessary. Esp as the boot jar stuff isn't needed for the project creation stuff.

Reply:
I will raise a story to bump the version of the galasa boot jar I guess.

Reply:
There is no 0.25.0 version of the boot jar. That's the latest. 
I don't have a problem with the boot jar being at whatever level. If we need to change it then we should bump the level though.

You think we should do that in v0.26.0 ? If so, perhaps you could raise a story ?